NEWS.md

```diff
@@ -1,3 +1,30 @@
+# dados 0.2.0
+
+## Major changes
+
+* New dataset: translation of `tidyr::billboard`.
+
+* New dataset: translation of `tidyr::household`.
+
+* New dataset: translation of `tidyr::who2`.
+
+* New dataset: translation of `tidyr::cms_patient_experience`.
+
+* New dataset: adaptation of `stringr::fruit`.
+
+* New dataset: adaptation of `stringr::sentences`.
+
+* New dataset: adaptation of `stringr::words`.
+
+## Minor changes
+
+* On the dataset `voos`, column `cauda` was renamed to `codigo_cauda`, so it's consistent with the dataset `avioes`.
+
+* Add Eric Scopinho as co-author.
```

Comment:
New authors are not announced on NEWS. This is only for changes that affect how people use the package